components/schemas/scoped-variables/ScopedVariableAccess.yml

```diff
@@ -0,0 +1,25 @@
+title: ScopedVariableAccess
+type: object
+description: The type or way the scoped variable is accessed.
+required:
+  - env_variable
+properties:
+  env_variable:
+    type: boolean
+    description: A boolean, where true represents this scoped variable is accessed like a normal environment variable.
+  internal_api:
+    nullable: true
+    type: object
+    properties:
+      duration:
+        $ref: ../Duration.yml
```

Comment:
This is also nullable. You need to wrap it with an anyOf structure like I do in several other places to utilize both the $ref and 'nullable'. This won't be an issue when we can finally upgrade to 3.1.

![image](https://github.com/cycleplatform/api-spec/assets/5110855/b9954a79-2708-4067-9e62-8bf21c137081)

Reply:
Also this needs a description - you can merge a description and a $ref tho so shouldn't be an issue.